fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/CollectionRobot.kt

```diff
@@ -68,12 +75,14 @@ class CollectionRobot {
     }
 
     fun saveTabsSelectedForCollection() {
-        mDevice.findObject(UiSelector().resourceId("$packageName:id/save_button")).click()
+        itemWithResId("$packageName:id/save_button").click()
+        Log.i(TAG, "saveTabsSelectedForCollection: Clicked \"Save\" button")
     }
 
     fun verifyTabSavedInCollection(title: String, visible: Boolean = true) {
         if (visible) {
             scrollToElementByText(title)
+            Log.i(TAG, "verifyTabSavedInCollection: Scrolled to: $title saved tab")
```

Comment:
As this is a custom function, you could actually add a log inside the scroll function.

Reply:
Done !